code/JAR/jackalope_v2.py

```diff
@@ -22,24 +23,57 @@ def pop_increase():
     for i in population: 
         if i['pregnant'] == True:
             jack.append(birth())
+        i['pregnant'] = False
     return jack
 
+# Age check for females
```

Comment:
These situations where a comment, created by a `#` symbol, above a function is a great opportunity to use `docstrings` instead. The docstring is a way to describe what a function does, what arguments are accepted, and what it returns. Check out how to use them. They are super simple but can be very useful when you are referencing the function elsewhere. See me if you're interested in a demo.